packages/eslint-plugin-shopify/lib/config/rules/babel.js

```diff
@@ -2,22 +2,12 @@
 // also turned off in configs that include these rules.
 
 module.exports = {
-  // Handles destructuring arrays with flow type in function parameters
-  'babel/array-bracket-spacing': ['warn', 'never'],
-  // Handles async functions correctly
-  'babel/arrow-parens': ['warn', 'always'],
-  // Require a particular separator between properties in Flow object types.
-  'babel/flow-object-type': ['warn', 'comma'],
-  // Require or forbid trailing commas for function paramater lists.
-  'babel/func-params-comma-dangle': ['warn', 'always-multiline'],
-  // Handles async/await functions correctly
-  'babel/generator-star-spacing': ['warn', 'after'],
```

Comment:
Can you move all this configuration stuff to the matching non-Babel rules? I think we turned off all of these from core ESLint when we needed this plugin to lint them with some newer ES features.